core/src/main/java/org/infinispan/reactive/publisher/impl/ClusterPublisherManagerImpl.java

```diff
@@ -987,7 +987,7 @@ public CompletionStage<Long> sizePublisher(IntSet segments, InvocationContext ct
        */
       private <E> Flowable<E> getValuesFlowable(BiFunction<InnerPublisherSubscription.InnerPublisherSubscriptionBuilder<K, I, R>, Map.Entry<Address, IntSet>, Publisher<E>> subToFlowableFunction) {
          return Flowable.defer(() -> {
-            if (!componentRegistry.getStatus().allowInvocations()) {
+            if (!componentRegistry.getStatus().allowInvocations() && !componentRegistry.getStatus().startingUp()) {
```

Comment:
So I assume the topology exception is thrown later in this case?

Reply:
Yep, once it reaches the `InvocationContextInterceptor`. I am unsure if this is one of the places we could fail earlier and avoid creating/sending all requests.